components/forecast/WeatherTab.tsx

```diff
@@ -144,16 +153,23 @@ export const WeatherTab: React.FC<WeatherTabProps> = ({zone}) => {
               <Body>Error loading weather stations.</Body>
             </HStack>
           )}
-          {weatherStationStatus === 'success' &&
-            groupedWeatherStations.length > 0 &&
-            groupedWeatherStations.map(([name]) => (
-              <HStack key={name} borderBottomWidth={1} borderColor={colorLookup('light.200')} py={8}>
-                <VStack>
-                  <Body>{name}</Body>
-                  {/* <BodyXSm>stations: {stations.map(s => s.stid).join(',')}</BodyXSm> */}
-                </VStack>
-              </HStack>
-            ))}
+          {weatherStationStatus === 'success' && groupedWeatherStations.length > 0 && (
+            <ActionList
+              actions={groupedWeatherStations.map(([name, stations]) => ({
+                label: name,
+                data: stations,
+                action: () => {
+                  // Nested navigation to the stationDetail page of the Weather Data stack
+                  navigation.navigate('Weather Data', {
+                    screen: 'stationDetail',
+                    // Treat this as the first screen in the Weather Data stack - don't show a back button going to the stationList
```

Comment:
How does the user get back to the weather tab?

Reply:
We talked about this a bit in the design meeting, there isn't a design yet. Currently you have to tap the home icon in the tab bar (so you're not stuck).

Reply:
Oh sorry, I had understood it to mean there were back arrows to let you get back to the forecast, e.g. weather station -> weather tab / forecast tab view